.github/workflows/pr-e2e.yml

```diff
@@ -158,6 +176,18 @@ jobs:
           commentId: ${{ github.event.comment.id }}
           reaction: "+1"
 
+      - uses: actions/github-script@v6
+        if: steps.test.outcome == 'success'
+        name: Add ok-to-merge label
+        with:
+          script: |
+            github.rest.issues.addLabels({
+              issue_number: ${{ needs.triage.outputs.pr_num }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              labels: ['ok-to-merge']
```

Comment:
Idea: Move label name to env var so that it's defined in one place and we can easily change it

Reply:
Good idea, let me check how to do it with GH actions

Reply:
what do you mean with in one place? as a secret for instance, that can be shared across all the workflows?

Reply:
No, just as an env var in this workflow.